autogluon/task/tabular_prediction/predictor.py

```diff
@@ -315,6 +315,42 @@ def fit_summary(self, verbosity=3):
             print("*** End of fit() summary ***")
         return results
 
+    # TODO: Consider adding time_limit option to early stop the feature importance process
+    def feature_importance(self, model=None, dataset=None, features=None, raw=True, silent=False):
+        """
+        Calculates feature importance scores for the given model.
+        Feature scores represent the score reduction the model achieves when the provided data's feature is randomly shuffled among its rows.
```

Comment:
Should we change:

"Feature scores represent the score reduction the model achieves when the provided data's feature is randomly shuffled among its rows."
->
"A feature's importance score represents the performance drop that results when the model makes predictions on a perturbed copy of the dataset where this feature's values have been randomly shuffled across rows"

Reply:
Updated